mobile2/src/screens/Books.tsx

```diff
@@ -2,14 +2,23 @@ import * as React from "react";
 import { createMaterialTopTabNavigator } from "@react-navigation/material-top-tabs";
 import { FinishedReading } from "./FinishedReading";
 import { ToBeRead } from "./Tbr";
+import { styles } from "../constants";
+import { FAB } from "react-native-paper";
 
 const Tab = createMaterialTopTabNavigator();
 
-export function Books() {
+export function Books({ navigation }) {
```

Comment:
There will be a type error here, but let's ignore it

Reply:
Yaa, I'll add the interface if I have time left.